.github/workflows/integration-testing.yml

```diff
@@ -50,13 +54,19 @@ jobs:
           key: ${{ runner.os }}-buildx-${{ github.sha }}
           restore-keys: |
             ${{ runner.os }}-buildx-
-      - name: Build docker image
+      - name: Create unique combination using matrix + commit parameters
+        run: |
+          echo "MATRIX_COMMIT_COMBO=${{ matrix.app-platform }}-${{ matrix.instrumentation-type }}-${{ github.sha }}" | tee --append $GITHUB_ENV;
+      - name: Construct Sample App image tag
+        run: |
+          echo "APP_IMAGE=ghcr.io/${{ github.repository }}-sample-app-${{ env.MATRIX_COMMIT_COMBO }}:latest" | tee --append $GITHUB_ENV;
```

Comment:
A slash needs to follow the repository so it's scoped to this repository.

Reply:
Cool thanks for that, changes made.